lib/node_modules/@stdlib/regexp/to-json/benchmark/benchmark.js

```diff
@@ -0,0 +1,49 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2022 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+'use strict';
+
+// MODULES //
+
+var bench = require( '@stdlib/bench' );
+var pkg = require( './../package.json' ).name;
+var toJSON = require( './../lib' );
+
+
+// MAIN //
+
+bench( pkg, function benchmark( b ) {
+	var regexp;
+	var o;
+	var i;
+
+	b.tic();
+	regexp = /beep/;
+	for ( i = 0; i < b.iterations; i++ ) {
+		o = toJSON( regexp );
+		if ( typeof o !== 'object' ) {
+			b.fail( 'should return an object' );
+		}
+	}
+	b.toc();
+	if ( typeof o !== 'object' ) {
+		b.fail( 'should return an object' );
+	}
+	b.pass( 'benchmark finished' );
+	b.end();
```

Comment:
```suggestion
	var values;
	var o;
	var i;
	
	values = [
		/beep/,
		/boop/,
		/.*/,
		/ab+c/
	];

	b.tic();
	for ( i = 0; i < b.iterations; i++ ) {
		o = toJSON( values[ i%values.length ] );
		if ( typeof o !== 'object' ) {
			b.fail( 'should return an object' );
		}
	}
	b.toc();
	if ( typeof o !== 'object' ) {
		b.fail( 'should return an object' );
	}
	b.pass( 'benchmark finished' );
	b.end();
```

Moving RegExp creation to outside the `tic`/`toc` block and using a list of values to prevent certain compiler optimizations.